Lesson16/Merve/script.js

```diff
@@ -0,0 +1,129 @@
+// 3. Write a function that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
+function compareNumbers(num1, num2) {
```

Comment:
This is a function declaration. Unless you call the function, it will never actually "work". So after the closing '}' of the function (line 9 currently), add e.g. `compareNumbers(1, 5)` (any number combination would work) or even better `console.log(compareNumbers(1, 5))` to see the output on the console. Make sure to do that for every exercises to verify that the function produces correct result.